.github/workflows/tests.yml

```diff
@@ -79,22 +128,20 @@ jobs:
     strategy:
       max-parallel: 4
       matrix:
-        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11']
+        python-version: ['3.8', '3.9', '3.10', '3.11']
         os: [ubuntu-latest]
         include:
           - python-version: '3.11'
             jupyter_server-version: '<2'
           - python-version: '3.11'
             jupyter_server-version: '>=2'
-          - python-version: '3.7'
-            os: windows-latest
```

Comment:
I think this would be good to restore but bumping it to 3.8.

Reply:
I am not sure to get what it means.

Reply:
I mean restore the windows test at the lower end of the supported versions which now is 3.8.

Reply:
My understanding is that the original intent here was: test on Linux across all versions and on Windows at the oldest and at the most recent. I think this is a good pattern in general.

Reply:
See https://github.com/jupyter/nbdime/pull/664/files#r1314193757